git-hooks/pre-push

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
-files=$(git diff --cached --name-only --diff-filter=ACM main)
+files=$(git diff --cached --name-only --diff-filter=ACM main | grep '\.js$')
 
 # Check for debugger statements in JavaScript files
-lines=$(grep -n 'debugger' $files)
+lines=$(grep -n 'debugger;' $files)
```

Comment:
The git hook wouldn't let me push this branch because the README.md mentioned "debugger" in it. So, I made this apply only to js files and only when it is "debugger;"  That's why this had to be included here.